setup.yml

```diff
@@ -149,6 +151,12 @@
         - export GOPATH=/root/.go
         - PATH=$GOPATH/bin:$PATH
 
+    - name: alternative go link created
```

Comment:
This needs to be conditional on `when: ansible_distribution == "Ubuntu"`

Reply:
done